package/yast2-ruby-bindings.spec

```diff
@@ -99,6 +99,7 @@ cd -
 
 %files
 %defattr (-, root, root)
+%{yast_ybindir}/y2base
```

Comment:
No, this way we forget it. Add it right away with the currently correct version.